hazelcast-jet-core/src/main/java/com/hazelcast/jet/Processors.java

```diff
@@ -90,7 +90,55 @@ public static ProcessorMetaSupplier writeMap(@Nonnull String mapName) {
      */
     @Nonnull
     public static ProcessorMetaSupplier writeMap(@Nonnull String mapName, @Nonnull ClientConfig clientConfig) {
-        return WriteIMapP.supplier(mapName, clientConfig);
+        return WriteBufferedP.writeMap(mapName, clientConfig);
+    }
+
+    /**
+     * Returns a meta-supplier of processors that will fetch entries from the
+     * Hazelcast {@code ICache} with the specified name. The processors will only
+     * access data local to the member and, if {@code localParallelism} for the
+     * vertex is above one, processors will divide the labor within the member
+     * so that each one gets a subset of all local partitions to read.
+     * <p>
+     * The number of Hazelcast partitions should be configured to at least
+     * {@code localParallelism * clusterSize}, otherwise some processors will have
+     * no partitions assigned to them.
+     * <p>
+     * If the underlying cache is concurrently being modified, there are no guarantees
+     * given with respect to missing or duplicate items.
+     */
+    @Nonnull
+    public static ProcessorMetaSupplier readCache(@Nonnull String cacheName) {
+        return ReadWithPartitionIteratorP.readCache(cacheName);
+    }
+
+    /**
+     * Returns a meta-supplier of processors that will fetch entries from a
+     * Hazelcast {@code ICache} in a remote cluster.
+     * <p>
+     * If the underlying cache is concurrently being modified, there are no guarantees
+     * given with respect to missing or duplicate items.
+     */
+    @Nonnull
+    public static ProcessorMetaSupplier readCache(@Nonnull String cacheName, @Nonnull ClientConfig clientConfig) {
+        return ReadWithPartitionIteratorP.readCache(cacheName, clientConfig);
+    }
+
+    /**
+     * Returns a meta-supplier of processors that will put data into a Hazelcast {@code ICache}.
+     */
+    @Nonnull
+    public static ProcessorMetaSupplier writeCache(@Nonnull String cacheName) {
+        return WriteBufferedP.writeCache(cacheName);
+    }
+
+    /**
+     * Returns a meta-supplier of processors that will put data into a Hazelcast {@code ICache} in
+     * a remote cluster.
+     */
+    @Nonnull
+    public static ProcessorMetaSupplier writeCache(@Nonnull String cacheName, @Nonnull ClientConfig clientConfig) {
```

Comment:
likewise, can we add that it expects entries of type `Map.Entry`/`Cache.Entry` to the docs?